README.md

```diff
@@ -1,12 +1,23 @@
 # Drift Python Client
 
-Python Client to access data of Devices on Drift Platform
+Python Client to access data of [PANDA|Drift](docs/panda_drift.md)
 
 ## Description
 
-Drift Python Client is a high level library to get current or historical data with minimal knowledge about Drift
+Drift Python Client is a high level library to get current or historical data with minimal knowledge about **PANDA|Drift**
 infrastructure.
 
+## Features
+
+* Subscribe to MQTT data
```

Comment:
so it's an MQTT client or is it important that you can access "Drift-encoded MQTT data" or something?